src/pages/DelegatoryValidator/DetailCard.tsx

```diff
@@ -62,11 +63,22 @@ export default function ValidatorDetailCard({
           value={null}
           sx={sx}
         />
-        <ContentRow title="Compound Rewards" value={null} sx={sx} />
-        <ContentRow title="Operator Commission" value={null} sx={sx} />
+        <ContentRow
```

Comment:
looks like `container` is an optional prop, you might simply dont need to pass it here - so it would be false/undefined in the `ContentRow` component

Reply:
the current way of making it work is:
1. container is an optional boolean variable
1.1 if container's not specified, default to `true`, so that `container={container && true}` => `container={true}`.
1.2 if container's specified as `false`, so that `container={container && true}` => `container={false}`. which is what we want in `DetailCard` cuz `container={true}` will not make "flex and space-between" work.
1.3 if container's specified as `true`, same to 1.1

so the example above, if we don't pass in `container={false}`, `container` will equal to `true` in `ContentRow`, not the exp we want. 

so you may ask, why not the other way around. set it default to false, and don't pass in container variable. it will work to this specific use case, but then we need to change all the code pointers where `ContentRow` is used to have then pass in `container={true}` instead.

a bit complicated, lmk if you have any questions!